### ISSUE TITLE
Dados no PZ Server 

### ISSUE THREAD

**Issue by gschwend** [linea-it/ic-photoz#1]
## Disponibilizar dados e resultados no PZ Server 

- [x] Gerar um spec-z sample simulado (arquivo tabular, pode ser csv, fits, hdf5 ou parquet) com o resultado do join entre os random IDs e a tabela `truth` do DP0.2, contendo apenas os IDs, ra, dec, e true-z (e magnitudes truth, opcional). 
- [x] Fazer upload desse arquivo no PZ Server (product type = `Spec-z Catalog`).  
- [x] Adicionar a versão HTML do notebook utilizado para a criação desse arquivo contendo os outputs como ancillary file para descrição do data product criado e mostrando a distribuição N(z) (pode ser no momento do upload ou depois).   
- [x] Gerar um training set simulado (outro arquivo tabular) completo (= antes dos cortes de qualidade e da separação `train`/`test`) a partir do join ou x-match  do conjunto de speczs acima com a tabela `objects` contendo IDs, ra, dec, true z, mags (observed), mag errs, outras colunas de interesse (opcional). 
- [x] Fazer upload desse arquivo no PZ Server (product type = `Training Set`). 
- [x] Adicionar a versão HTML do notebook como ancillary file para descrição do data product criado, contendo N(mag), N(cores), N(z), mag X err, CMD, color-color (QA completo).  
- [x] Gerar um .tar file com os arquivos de resultado da validação do PZ. 
- [x] Fazer upload desse arquivo .tar no PZ Server (product type = `Photo-z Validation Results`)  
    Conteúdo: 
    - arquivo tabular com as pdfs do test set 
    - outros arquivos relevantes (ex: array de random IDs, arquivos de configuração, etc) 
    - versão HTML do notebook que foi utilizado e que garanta a reprodutibilidade dos passos: 
        * cortes de qualidade
        * separação do training set em dois subsets sem repetição de elementos: `train` e `test` 
        * treinamento do algoritmo
        * estimativa das pdfs 
        * cáclulo das métricas

**Comment by gschwend** [comment]
@andreiadourado os arquivos de descrição para os data products gerados no upload do PZ Server podem ser editados no futuro, não espere ter uma versão definitiva deles para fazer os uploads.
